policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,45 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products. This policy describes the lifecycle of feature development.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprised of six phases. 
+
+1. Requirements Gathering and Planning:
+  - Identify the scope, goals, and necessary resources for development
+  - Define functional and security requirements for the feature 
+    - Considering [core engineering principles](https://github.com/MetaMask/contributor-docs/blob/main/policies/engineering-principles.md)
+    - Security controls such as authentication, data encryption, and secure communication protocols
+
+2. Design:
+- Create a feature description that incorporates identified security controls and meets [definition of ready](https://www.notion.so/What-is-the-Definition-of-Ready-fa1df34b96734791be1293d4f536869c)
```

Comment:
On the subject of security: Considering this repo is public, are we risking anything by linking out to a page in our private Notion? Should we copy this to a new document in this repo? (same goes for the General Incident Response Playbook)

Reply:
@mcmire Good point. I will remove the notion links and we can move these documents into this repo in the future.